Xamarin.Forms.Core/RelativeLayout.cs

```diff
@@ -248,13 +267,13 @@ void CreateBoundsFromConstraints(View view, Constraint xConstraint, Constraint y
 		static Rectangle SolveView(View view)
 		{
 			BoundsConstraint boundsConstraint = GetBoundsConstraint(view);
-			var result = new Rectangle();
 
 			if (boundsConstraint == null)
 			{
 				throw new Exception("BoundsConstraint should not be null at this point");
 			}
-			result = boundsConstraint.Compute();
+
+			var result = boundsConstraint.Compute();
 
 			return result;
```

Comment:
I saw that too, but I didn't want to fix 2 unrelated things in one pull request.